### PR TITLE
fix: you cannot postpone and resume a zaak if the zaak has no streefdatum

### PR DESCRIPTION
**Pull request by edgarvonk**
Fixed code so that you can now postpone and resume a zaak when the zaak/zaaktype does not have a 'streefdatum', which may be the case depending on how the zaaktype is configured.

Solves PZ-1407

### CODE REVIEW

File: src/main/java/net/atos/zac/shared/helper/OpschortenZaakHelper.java
Comment (bas-info-nl) [inline src/main/java/net/atos/zac/shared/helper/OpschortenZaakHelper.java:94]: English and Nederlands mixed, does not make it easier to read...

**Review by bas-info-nl — APPROVED** [review]

**Comment by edgarvonk** [comment]
Note: on hold for now. Awaiting approval from PO.

**Pull request merged**
